gamemodes/terrortown/gamemode/server/sv_shopeditor.lua

```diff
@@ -63,7 +63,7 @@ function ShopEditor.CreateShopDBs()
 	for i = 1, #rlsList do
 		local rd = rlsList[i]
 
-		if rd == INNOCENT then continue end
+		if rd.index == ROLE_INNOCENT or rd.index == ROLE_NONE then continue end
```

Comment:
Now, this can be modified so Innocents could have an own shop (by default disabled)